M2/Macaulay2/m2/undoc.m2

```diff
@@ -1,4 +1,12 @@
 undocumented {
+    -- documented in MinimalPrimes
+    decompose,
+    -- documented in Colon
+    annihilator,
+    saturate,
+    -- documented in LocalRings
+    symbol MaximalIdeal,
+    -- etc.
```

Comment:
In principle I agree with you, but this of this as a slow progress towards moving pieces of Core into independent packages. Eventually when the Core doesn't call on saturate or annihilator, I will move the method declarations into Colon.

Reply:
I disagree, at least about decompose.  There are many uses of the method decompose which have nothing to do with minimal primes (e.g. decompose a betti table).  In fact, I would prefer to move other names to be defined in Core, rather than in each separate package (e.g. face, facet), as when using several packages together that define the same name, one has to use $'s all over the place, and when loading a package like that (I have one example, not in the system yet) one gets multiple warning messages about symbol being overloaded..., making the package look not polished.

Reply:
Good point.

Reply:
Defining `decompose` as a method in the core is one issue, but what about calling a method installed on it by another package from the core, as randomKRationalPoint does?  That seems unclean.  In particular, if you add `--no-preload` to the command line, that function will mysteriously stop working.

Reply:
I believe we do something like that for finding heft functions too, inside of `m2/ofcm.m2`, in `findHeft`, we do:
```
     needsPackage "FourierMotzkin";
     B := ((value getGlobalSymbol "fourierMotzkin") A)#0;
```

Reply:
You could use this instead:
```m2
B := importFrom("FourierMotzkin", {"fourierMotzkin"});
```

Why can't randomKRationalPoint move to RandomRationalPoints? It's not used anywhere else in the Core.

Reply:
Also, why don't we preload more packages? E.g. the ones that define graph, face, facet, cone, etc.